sagemaker-python-sdk/mxnet_onnx_eia/mxnet_onnx_eia.ipynb

```diff
@@ -0,0 +1,321 @@
+   "outputs": [],
+   "source": [
+    "import mxnet as mx\n",
+    "mx.test_utils.download('https://s3.amazonaws.com/onnx-model-zoo/resnet/resnet152v1/resnet152v1.onnx')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Compressing the model data\n",
+    "\n",
+    "Now that we have the model data locally, we will need to compress it and upload the tarball to S3 for the SageMaker Python SDK to create a Model"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import tarfile\n",
+    "\n",
+    "from sagemaker.session import Session\n",
+    "\n",
+    "with tarfile.open('onnx_model.tar.gz', mode='w:gz') as archive:\n",
+    "    archive.add('resnet152v1.onnx')\n",
+    "\n",
+    "model_data = Session().upload_data(path='onnx_model.tar.gz', key_prefix='model')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Creating a SageMaker Python SDK Model instance\n",
+    "\n",
+    "With the model data uploaded to S3, we now have everything we need to instantiate a SageMaker Python SDK Model. We'll provide the constructor the following arguments:\n",
+    "\n",
+    "* `model_data`: the S3 location of the model data\n",
+    "* `entry_point`: the script for model hosting that we looked at above\n",
+    "* `role`: the IAM role used\n",
+    "* `framework_version`: the MXNet version in use, in this case '1.3.0'\n",
+    "\n",
+    "You can read more about creating an `MXNetModel` object in the [SageMaker Python SDK API docs](https://sagemaker.readthedocs.io/en/latest/sagemaker.mxnet.html#mxnet-model)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from sagemaker.mxnet import MXNetModel\n",
+    "\n",
+    "mxnet_model = MXNetModel(model_data=model_data,\n",
+    "                         entry_point='resnet152.py',\n",
+    "                         role=role,\n",
+    "                         framework_version='1.3.0')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Creating an inference endpoint and attaching an Elastic Inference(EI) accelerator\n",
+    "\n",
+    "Now we can use our `MXNetModel` object to build and deploy an `MXNetPredictor`. This creates a SageMaker Model and Endpoint, the latter of which we can use for performing inference. \n",
+    "\n",
+    "We pass the following arguments to the deploy method:\n",
+    "\n",
+    "* instance_count - how many instances to back the endpoint.\n",
+    "* instance_type - which EC2 instance type to use for the endpoint. For information on supported instance, please check here.\n",
+    "* accelerator_type - determines which EI accelerator type to attach to each of our instances. The supported types of accelerators can be found here: https://aws.amazon.com/sagemaker/pricing/instance-types/\n",
```

Comment:
it's nice to put args, method names, etc. in backticks to make them monospace. I think there are a handful of instances throughout the notebook